sdk/identity/identity/package.json

```diff
@@ -43,7 +43,13 @@
   "repository": "github:Azure/azure-sdk-for-js",
   "keywords": [
     "azure",
-    "cloud"
+    "active directory",
+    "authentication",
+    "credential",
+    "certificate",
+    "managed service identity",
+    "client secret",
+    "access token"
```

Comment:
Does npm work well with keywords that have spaces? I have no clue, just asking :)

Reply:
There doesn't seem to be any specific rules for keywords in the [npm documentation](https://docs.npmjs.com/files/package.json#keywords) so I think spaces should be fine.  We can certainly revisit this if it turns out to be a problem.